ipatests/test_integration/test_sssd.py

```diff
@@ -286,3 +288,29 @@ def get_cache_update_time(obj_kind, obj_name):
         finally:
             sssd_conf_backup.restore()
             tasks.clear_sssd_cache(self.master)
+
+    def test_trustdomain_disable(self):
```

Comment:
This is related to trust specific test. Could you add this in test_trust.py
With this there will be no need to change in topology also.

Reply:
There are other AD-related tests in this test suite. SSSD does communicate with trusted domains, yes. 
But this particular test is for regression in sssd, not in IPA.

Reply:
Do we have any guidelines that regression with sssd should be in test_sssd.py.
With this we are adding test plus resources which we are already using in trust.py
And you can enable it again using `ipa trustdomain-enable` so it wont impact on other tests.

Reply:
1. There is a docstring at the beginning of the test suite. 
2. All sssd regression tests are already here, it would be nice to keep it consistent.

I would not worry about increased resources too much as this testsuite is not executed for gating, only for nightly runs so it generally does not affect testing time of the PRs.

Reply:
"ipa trustdomain-enable" is not needed since trust is removed at the end of the test.